### ISSUE TITLE
"Edit this page" fails for fetched loaders/plugins

### ISSUE THREAD

**Issue by simon04** [webpack/webpack.js.org#595]
Clicking "Edit this page" on https://webpack.js.org/plugins/i18n-webpack-plugin/ leads to a non existing file/page.

This affects all [fetched](https://github.com/webpack/webpack.js.org/blob/master/scripts/fetch.sh) loaders/plugins.

**Comment by bebraw** [comment]
Good point. I guess we should point to the package README editors then.
